db/migrate/20230220204405_delete_ovirt_rhv_network_event_catcher_workers.rb

```diff
@@ -0,0 +1,14 @@
+class DeleteOvirtRhvNetworkEventCatcherWorkers < ActiveRecord::Migration[6.1]
+  class MiqWorker < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
+  end
+
+  def up
+    MiqWorker.where(
```

Comment:
@agrare Can you wrap this in `say_with_time` ?

Reply:
Ah yeah forgot, updated